apps/api/v2/src/ee/bookings/2024-08-13/services/bookings.service.ts

```diff
@@ -805,4 +806,32 @@ export class BookingsService_2024_08_13 {
       t: await getTranslation("en", "common"),
     });
   }
+
+  async getBookingReferences(bookingUid: string, userId: number) {
```

Comment:
Just like with repository, I would suggest moving `getBookingReferences` and `getOrgBookingReferences` to `booking-references.service.ts` that is then invoked by the current endpoints. The bookings service is growing large and at this point its better to split it like this especially given that these functions call booking references repository / model.